custom_components/trakt_tv/schema.py

```diff
@@ -41,6 +41,7 @@ def sensors_schema() -> Dict[str, Any]:
         "all_upcoming": upcoming_schema(),
         "next_to_watch": next_to_watch_schema(),
         "recommendation": recommendation_schema(),
+        "lists": All([lists_schema()]),
```

Comment:
Not sure that you should add "All" here. Reading the documentation, I don't really know how to implement correctly though...

Reply:
`Schema([lists_schema()])` might be better here, will test it

Reply:
Yep would like to help more but voluptuous documentation is not clear